### ISSUE TITLE
[BUG]: github_emu_group_mapping - Invalid address to set: []string{"group"} in v6

### ISSUE THREAD

**Issue by gm-hyp-2** [integrations/terraform-provider-github#2131]
### Expected Behavior

Until v5.45.0 `github_emu_group_mapping` worked just fine. When upgrading to v6.0.0.0 and later, the resource cannot be used anymore. 

### Actual Behavior

When trying to run terraform plan on the resource github_emu_group_mapping: https://registry.terraform.io/providers/integrations/github/latest/docs/resources/emu_group_mapping it always breaks with this reason:
```
Error: Invalid address to set: []string{"group"}

  with github_emu_group_mapping.pure_fb_mapping["prague-object-one"],
  on teams.tf line 44, in resource "github_emu_group_mapping" "pure_fb_mapping":
  44: resource "github_emu_group_mapping" "pure_fb_mapping" {
```

### Terraform Version

Terraform 1.7.2 on linux_amd64
Affected versions v6.0.0.0-alpha, v6.0.0.0-beta and v6.0.0.0-rc1

### Affected Resource(s)

github_emu_group_mapping

### Steps to Reproduce

Normal tf plan will highlight the error

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kfcampbell** [comment]
This behavior should be fixed by #2133. In previous versions of the provider, we were ignoring that error. Can you please try [v6.0.0-rc2](https://github.com/integrations/terraform-provider-github/releases/tag/v6.0.0-rc2) and report back to see if that fixes the error?

**Comment by gm-hyp-2** [comment]
Yes, that PR seems to have resolved the issue. Thank you!
